build.gradle

```diff
@@ -15,14 +15,16 @@
 //
 
 plugins {
-  id 'java'
-  id 'maven-publish'
-  id 'signing'
-  id 'com.github.johnrengelman.shadow' version '7.0.0'
-  id 'net.nemerosa.versioning' version '2.14.0'
-  id 'io.github.gradle-nexus.publish-plugin' version '1.1.0'
-  id 'antlr'
+    id 'com.github.johnrengelman.shadow' version '7.0.0'
+    id 'net.nemerosa.versioning' version '2.14.0'
+    id 'io.codearte.nexus-staging' version '0.22.0'
+    id 'antlr'
 }
+apply plugin: 'java'
```

Comment:
Can't this go in the plugins { ... } block?

Reply:
done